actions/issue-reopener/src/main.ts

```diff
@@ -0,0 +1,179 @@
+    throw new ValidationError(filePath, expectedDigest, computedDigest);
+  }
+}
+
+// downloadSLSAVerifier downloads the slsa-verifier binary, verifies it against
+// the expected sha256 hex digest, and returns the path to the binary.
+export async function downloadSLSAVerifier(
+  path: string,
+  version: string,
+  digest: string,
+): Promise<string> {
+  // Download the slsa-verifier binary.
+  const verifierPath = await tc.downloadTool(
+    `https://github.com/slsa-framework/slsa-verifier/releases/download/${version}/slsa-verifier-linux-amd64`,
+    `${path}/slsa-verifier`,
+  );
+
+  // Validate the checksum.
+  await validateFileDigest(verifierPath, digest);
+
+  return verifierPath;
+}
+
+// downloadAndVerify downloads the `github-issue-reopener` binary and verifies
+// the associated SLSA provenance.
+export async function downloadAndVerify(
+  path: string,
+  version: string,
+  slsaVerifierVersion: string,
+  slsaVerifierDigest: string,
+): Promise<string> {
+  const verifierPath = await downloadSLSAVerifier(
+    path,
+    slsaVerifierVersion,
+    slsaVerifierDigest,
+  );
+
+  // Download the github-issue-reopener binary.
+  const reopenerPath = await tc.downloadTool(
+    `https://github.com/ianlewis/todos/releases/download/${version}/github-issue-reopener-linux-amd64`,
+    `${path}/github-issue-reopener`,
+  );
+
+  const provenancePath = await tc.downloadTool(
+    `https://github.com/ianlewis/todos/releases/download/${version}/github-issue-reopener-linux-amd64.intoto.jsonl`,
+    `${path}/github-issue-reopener.intoto.jsonl`,
+  );
+
+  const { exitCode, stdout, stderr } = await exec.getExecOutput(
+    `${verifierPath}`,
+    [
+      "verify-artifact",
+      reopenerPath,
+      "--provenance-path",
+      provenancePath,
+      "--source-uri",
+      "github.com/ianlewis/todos",
+      "--source-tag",
+      version,
+    ],
+  );
+  if (exitCode !== 0) {
+    throw new Error(
+      `unable to verify binary provenance.\nstdout: ${stdout}; stderr: ${stderr}`,
+    );
+  }
+
+  return reopenerPath;
+}
+
+// cleanup deletes the temporary files.
+async function cleanup(path: string): Promise<void> {
+  await io.rmRF(`${path}`);
+}
+
+async function run(): Promise<void> {
+  const wd = core.getInput("path", { required: true });
+  const token = core.getInput("token", { required: true });
+  const tmpDir = await fs.mkdtemp(
+    path.join(os.tmpdir(), "github-issue-reopener_"),
+  );
+
+  let reopenerPath;
+  try {
+    reopenerPath = downloadAndVerify(
+      tmpDir,
+      REOPENER_VERSION,
+      SLSA_VERIFIER_VERSION,
+      SLSA_VERIFIER_SHA256SUM,
+    );
+  } catch (error: unknown) {
+    const errMsg = error instanceof Error ? error.message : String(error);
+    core.setFailed(`failed to download github-issue-reopener: ${errMsg}`);
+    await cleanup(tmpDir);
+    return;
+  }
+
+  // Run the github-issue-reopener.
+  const { exitCode, stdout, stderr } = await exec.getExecOutput(
```

Comment:
Probably this error is coming from `getExecOutput`

https://github.com/ianlewis/actions-test/actions/runs/5910352239/job/16031862728
```
/home/runner/work/_actions/ianlewis/todos/reopener-action/actions/issue-reopener/webpack:/issue-reopener/node_modules/@actions/io/lib/io.js:171
                    throw new Error(`Unable to locate executable file: ${tool}. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.`);
^
Error: Unable to locate executable file: [object. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
    at Object.<anonymous> (/home/runner/work/_actions/ianlewis/todos/reopener-action/actions/issue-reopener/webpack:/issue-reopener/node_modules/@actions/io/lib/io.js:1[7](https://github.com/ianlewis/actions-test/actions/runs/5910352239/job/16031862728#step:3:8)1:1)
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/_actions/ianlewis/todos/reopener-action/actions/issue-reopener/webpack:/issue-reopener/node_modules/@actions/io/lib/io.js:24:1)
```